modules/installation-special-config-encrypt-disk-tang.adoc

```diff
@@ -45,15 +45,14 @@ The advertisement contains the following signing keys:
 PLjNyRdGw03zlRoGjQYMahSZGu9
 ----
 
-.. When the `Do you wish to trust these keys? [ynYN]` prompt displays, type `Y`, and the thumbprint is displayed:
+Alternatively, run the following command directly on the Tang server:
```

Comment:
What's this command the alternative to? All of the process that starts at line 29?

Are there different use cases that would make a user pick one of the other?

Is this step really no longer necessary?

Either the nesting of the steps needs to change to present them as equal alternatives, or we need to present the one that we want users to actually do.

Reply:
this is an alternative to the `Gather the thumbprint from the Tang server` proposed workflow so i nested it accordingly